extensions/spatial.sigmf-ext.md

```diff
@@ -0,0 +1,362 @@
+north that is detecting emissions from an emitter located east north-east from
+the aperture (see figure above for reference).
+
+```json
+{
+  "global": {
+    "core:datatype": "ci16_le",
+    "core:sample_rate": 40000000,
+    "antenna:gain": 6,
+    "spatial:num_elements": 6
+  },
+  "captures": [
+    {
+      "core:sample_start": 0,
+      "core:frequency": 1260000000.0,
+      "spatial:element_geometry": [
+        { "point": [0, 0.1, 0.05] },
+        { "point": [0,   0, 0.05] },
+        { "point": [0,-0.1, 0.05] },
+        { "point": [0, 0.1,-0.05] },
+        { "point": [0,   0,-0.05] },
+        { "point": [0,-0.1,-0.05] }
+      ],
+      "spatial:aperture_bearing": {
+        "azimuth": 1.13124
+      }
+    }
+  ],
+  "annotations": [
+    {
+      "core:sample_start": 8424351,
+      "core:sample_count": 88741,
+      "core:freq_upper_edge": 1248111918.0,
+      "core:freq_lower_edge": 1245718776.0,
+      "core:description": "burst",
+      "spatial:signal_bearing": {
+        "azimuth": 59.431,
+        "elevation": 13.681,
+        "az_error": 1.3421,
+        "el_error": 4.1192
+      }
+    },
+    {
+      "core:sample_start": 13843284,
+      "core:sample_count": 96438,
+      "core:freq_upper_edge": 1271283241.0,
+      "core:freq_lower_edge": 1268532007.0,
+      "core:description": "burst",
+      "spatial:signal_bearing": {
+        "azimuth": 60.994,
+        "elevation": 17.324,
+        "az_error": 0.9694,
+        "el_error": 3.8474
+      }
+    },
+    ...
+  ]
+}
+```
+
+Here is an example of a 4 element aperture with square geometry in the XY plane:
+
+```json
+{
+  "global": {
+    "core:datatype": "ci16_le",
+    "core:sample_rate": 40000000,
+    "antenna:gain": 6,
+    "spatial:num_elements": 6
+  },
+  "captures": [
+    {
+      "core:sample_start": 0,
+      "core:frequency": 160000000.0,
+      "spatial:element_geometry": [
+        { "point": [ 0.25, 0.25,0] },
+        { "point": [ 0.25,-0.25,0] },
+        { "point": [-0.25,-0.25,0] },
+        { "point": [-0.25, 0.25,0] }
+      ],
+    }
+  ],
+  "annotations": [
+    {
+      "core:sample_start": 38012637,
+      "core:sample_count": 100991,
+      "core:freq_upper_edge": 165125034.0,
+      "core:freq_lower_edge": 165112388.0,
+      "spatial:signal_azimuth": 202.812
+    },
+    {
+      "core:sample_start": 780208811,
+      "core:sample_count": 100018,
+      "core:freq_upper_edge": 165125018.0,
+      "core:freq_lower_edge": 165112371.0,
+      "spatial:signal_azimuth": 200.142
+    },
+    {
+      "core:sample_start": 118009143,
+      "core:sample_count": 99841,
+      "core:freq_upper_edge": 165125041.0,
+      "core:freq_lower_edge": 165112369.0,
+      "spatial:signal_azimuth": 197.681
+    },
+    {
+      "core:sample_start": 158007123,
+      "core:sample_count": 101041,
+      "core:freq_upper_edge": 165125023.0,
+      "core:freq_lower_edge": 165112401.0,
+      "spatial:signal_azimuth": 195.017
+    },
+    ...
+  ]
+}
+```
```

Comment:
the alpha =0 background of the .png seems to hinder the visibility of the axis labels for me, using a dark theme.